src/kitovu/sync/plugin/moodle.py

```diff
@@ -79,7 +98,9 @@ def _list_files(self, course_path: pathlib.PurePath) -> typing.Iterable[str]:
                     if 'mimetype' not in elem:
                         continue
                     local_path = module_path / elem['filename']
```

Comment:
This now creates the files like: `/[section name]/[module name]/[filename]` where module is a sub-group of the sections.

Unfortunately the sections are not sorted as in the moodle, but alphabetically. The only possibility I see would be to prefix a order number, but that would re-download everything if the order would be changed....